src/main/java/com/recycle/buddy/service/ImageRecognitionService.java

```diff
@@ -0,0 +1,18 @@
+package com.recycle.buddy.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class ImageRecognitionService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ImageRecognitionService.class);
+
+    public String recognize(final String image) {
```

Comment:
Are we sure that this parameter will be of type String?

Reply:
No, we are not sure. I am still waiting an answer from Frond end.